RELEASING.md

```diff
@@ -44,12 +40,10 @@ ossrhPassword=<YOUR-NEXUS-PASSWORD>
 
 Deploy to Sonatype
 ------------------
-* Update all ```pom.xml``` files in the package to the release version you want. Sumit a pull request, get it reviewed, and submit
-* ```mvn clean install deploy -DperformRelease=true```
-* Verify the publishment [here](https://oss.sonatype.org/#nexus-search;quick~gax)
-  * If there is a problem, undo by ```mvn nexus-staging:drop```
-* ```mvn nexus-staging:release -DperformRelease=true```
-* Update all ```pom.xml``` files to the new snapshot version
+* Update the ```gax/version.txt``` and ```gax-grpc/version.txt``` files to the
+  release version you want
+* Run ```./gradlew stageRelease``` to update ```README.md``` and ```samples/pom.xml```
```

Comment:
The `stageRelease` task does more than this - maybe make a bullet for each thing it does (updates those files, generates javadocs, stages the artifact on Sonatype)